tests/auxiliary/minicore.rs

```diff
@@ -101,10 +102,101 @@ macro_rules! concat {
         /* compiler built-in */
     };
 }
+
 #[rustc_builtin_macro]
 #[macro_export]
 macro_rules! stringify {
     ($($t:tt)*) => {
         /* compiler built-in */
     };
 }
+
+#[macro_export]
+macro_rules! partialEq {
+    ($($ty:ty),* $(,)?) => {
+        $(
+            impl PartialEq for $ty {
+                #[inline]
+                fn eq(&self, other: &$ty) -> bool {
+                    (*self) == (*other)
+                }
+                #[inline]
+                fn ne(&self, other: &$ty) -> bool {
+                    (*self) != (*other)
+                }
+            }
+        )*
+    }
+}
+
+partialEq!(
+    bool, char, isize, i8, i16, i32, i64, i128, usize, u8, u16, u32, u64, u128, f16, f32, f64, f128
+);
+
+#[lang = "panic"]
+//#[rustc_const_panic_str]
+//#[inline(never)]
+//#[cold]
+//#[track_caller]
+//#[rustc_nounwind]
+const fn panic(expr: &'static str) -> ! {
+    abort()
+}
```

Comment:
Problem: let's pick `not(feature = "panic_immediate_abort")` configuration, and to match `core` (as best as possible) https://github.com/rust-lang/rust/blob/b522e7c5ea8334fbccb9dd7f9b33759304743fd0/library/core/src/panicking.rs#L128-L133

```rs
#[inline(never)]
#[cold]
#[track_caller]
#[lang = "panic"]
pub const fn panic(expr: &'static str) -> ! {
```